test/src/org/labkey/test/tests/targetedms/TargetedMSChromatogramOptimizationTest.java

```diff
@@ -0,0 +1,126 @@
+package org.labkey.test.tests.targetedms;
+
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.categories.MS2;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.targetedms.ConnectionSource;
+
+import java.io.File;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Category({DailyB.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 6)
+public class TargetedMSChromatogramOptimizationTest extends TargetedMSTest
+{
+    private static final String SKY_FILE = "SmallMoleculeLibrary3.sky.zip";
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        TargetedMSChromatogramOptimizationTest init = (TargetedMSChromatogramOptimizationTest) getCurrentTest();
+        init.setupFolder(FolderType.Library);
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return "TargetedMS Chromatogram Optimization Test";
+    }
+
+    @Test
+    public void testUpload() throws Exception
+    {
+        goToProjectHome();
+        importData(SKY_FILE);
+        clickAndWait(Locator.linkContainingText("Panorama Dashboard"));
+        File downloadedClibFile = doAndWaitForDownload(() -> clickButton("Download", 0));
+
+        goToProjectHome();
+        goToSchemaBrowser();
+        DataRegionTable table = viewQueryData("targetedms", "TransitionOptimization");
+        table.rowSelector().showAll();
+
+        checker().verifyEquals("Invalid number of rows in transition optimization", table.getDataRowCount(),
```

Comment:
This ordering seems odd. First we check the number of rows in the table, and then we separately check that the table exists?

Checking a few other tables would be good, as this test coverage is new for the generated SQLite files so there's some catchup to do.

Reply:
I am verifying the tables mentioned in the spec, Do u know any other table to be verified?

Reply:
See here for the schema:

https://docs.google.com/document/d/1U1Oiy6LaWRqd5EaJ0h-WHFTLrWg360yUdcXAj4XumzY/edit#heading=h.sxp2n0h84m9u

Or you can use a SQLite file browser (including IntelliJ) to crack open the .clib and see what it contains. This is for PyCharm instead of IntelliJ but the same drag/drop works great in both.

https://www.jetbrains.com/pycharm/guide/tips/create-sqlite-connection/

Reply:
added coverage based on the email reply.

Reply:
@labkey-sweta Please push your changes. They're not showing in GitHub yet.

Reply:
My push was rejected because the my branch was behind, should be there now.